src/hotspot/os/linux/cgroupV2Subsystem_linux.cpp

```diff
@@ -140,21 +146,25 @@ jlong CgroupV2Subsystem::memory_max_usage_in_bytes() {
 }
 
 jlong CgroupV2Subsystem::rss_usage_in_bytes() {
-  GET_CONTAINER_INFO_LINE(julong, _memory->controller(), "/memory.stat",
-                          "anon", JULONG_FORMAT, JULONG_FORMAT, rss);
-  return rss;
+  julong rss;
+  bool is_ok = _memory->controller()->
+                    read_numerical_key_value("/memory.stat", "anon", &rss);
+  if (!is_ok) {
```

Comment:
Style, nit: A bit unconventional (for Hotspot) to line break after `->`, maybe this is preferable:

```c++
bool is_ok =
  _memory->controller()->read_numerical_key_value("/memory.stat", "anon", &rss);
```

Reply:
Or each argument can be line-separated, just a couple of ideas.